src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Views/EmployerRequest/Dashboard.cshtml

```diff
@@ -118,9 +118,9 @@
                                     </div>
                                 </p>
                                 <p>
-                                    <div class="govuk-body govuk-!-margin-bottom-1">Request made on @request.RequestedAt.ToString("d MMM yyyy")</div>
+                                    <div class="govuk-body govuk-!-margin-bottom-1">Request made on @request.RequestedAt.ToString("d MMMM yyyy")</div>
```

Comment:
Even though the story doesn't say I think it's that all uses of MMM need to be converted to MMMM - like in other pages - but confirm that